### DISCUSSION TITLE
Would you still consider this package a good choice despite the low activiy here?

### DISCUSSION THREAD

**Discussion by bennylope** [bennylope/django-organizations#249] [Q&A]
From #248 by @tommueller 

> Hi,
> 
> looking for an honest opinion from some project members. I am seeking an organization management package for Django. As far as my research today went, I only found this single one here. It covers 95% of our requirements, but repo activity and community activity seems to be very low.
> 
> Is this just working and stable and all the ToDo's in the Wiki will not be done, but the repo is still actively maintained? Or should I rather invest into developing this ourselves, since nothing is happening here anymore?
> 
> Thanks!
> Tom

**Comment by bennylope** [comment]
Hi @tommueller, that wiki is very much outdated and irrelevant, now removed.

As far as project activity and status: as the original author and primary maintainer I am committed to ensuring this project is **available with continued support for subsequent Python and Django versions** and resolving bugs to the best of my ability. Feature development has been quiet for some time because the app largely does what I needed it to do, however I'm _very much_ open to new contributions (provided they're backwards compatible, etc.)!

**Comment by tomdtp** [comment]
Hi @bennylope, thanks for your quick response!

I will do some testing this week regarding the integration of this package and might get back here with some questions :) One possible for us could be https://github.com/bennylope/django-organizations/issues/243, this is still not possible, right?
